source/codegen/metadata/niscope/attributes.py

```diff
@@ -1,2556 +1,1558 @@
         'name': 'P2P_DESTINATION_WINDOW_ADDR',
         'resettable': True,
         'type': 'ViInt64'
     },
     1150332: {
-        'access': 'read-write',
-        'channel_based': False,
         'codegen_method': 'public',
-        'documentation': {
-            'description': '\nSpecifies the type of the NISCOPE_ATTR_P2P_DESTINATION_WINDOW_ADDR attribute.\nValid Values:\nPhysical (0)\nVirtual (1)\nDefault Value: Virtual\n',
-            'note': 'This attribute can be used only with high-speed digitizers that support peer-to-peer streaming.'
-        },
         'enum': 'AddressType',
-        'lv_property': 'Peer-to-Peer:Manual:Configuration:Destination Window Address Type',
+        'grpc_type': 'sint32',
         'name': 'P2P_DESTINATION_WINDOW_ADDR_TYPE',
         'resettable': True,
         'type': 'ViInt32'
     },
     1150333: {
-        'access': 'read-write',
-        'channel_based': False,
-        'codegen_method': 'public',
-        'documentation': {
-            'description': '\nSpecifies the size, in bytes, of the destination window determined by the NISCOPE_ATTR_P2P_DESTINATION_WINDOW_ADDRESS and the NISCOPE_ATTR_P2P_DESTINATION_WINDOW_ADDRESS_TYPE attributes.\nValid Values: Any non-NULL value.\n',
-            'note': 'This attribute can be used only with high-speed digitizers that support peer-to-peer streaming.'
-        },
-        'lv_property': 'Peer-to-Peer:Manual:Configuration:Destination Window Size',
+        'codegen_method': 'public',
+        'grpc_type': 'int64',
         'name': 'P2P_DESTINATION_WINDOW_SIZE',
         'resettable': True,
         'type': 'ViInt64'
     },
     1150334: {
-        'access': 'read-write',
-        'channel_based': False,
         'codegen_method': 'public',
-        'documentation': {
-            'description': '\nSpecifies the event to push the NISCOPE_ATTR_P2P_NOTIFY_MESSAGE_PUSH_VALUE attribute to the NISCOPE_ATTR_P2P_NOTIFY_MESSAGE_PUSH_ADDR attribute. Setting this attribute to NISCOPE_VAL_NOTIFY_DONE pushes the message when the acquisition has completed.\nValid Values:\nNever (0)\nDone (1)\nDefault Value: Done\n',
-            'note': 'This attribute can be used only with high-speed digitizers that support peer-to-peer streaming.'
-        },
         'enum': 'NotificationType',
-        'lv_property': 'Peer-to-Peer:Manual:Notification:Push Message On',
+        'grpc_type': 'sint32',
         'name': 'P2P_NOTIFY_PUSH_MESSAGE_ON',
         'resettable': True,
         'type': 'ViInt32'
     },
     1150335: {
-        'access': 'read-write',
-        'channel_based': False,
         'codegen_method': 'public',
-        'documentation': {
-            'description': '\nSpecifies the address to Push Message push Value to on the event specified by the NISCOPE_ATTR_P2P_NOTIFY_PUSH_MESSAGE_ON attribute.\n',
-            'note': 'This attribute can be used only with high-speed digitizers that support peer-to-peer streaming.'
-        },
-        'enum': 'AddressType',
```

Comment:
Missing 'enum' tag for this attribute. You can look [here](https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/358221?_a=files&path=/src/nidmm/nidmmBaseDistribution/nidmmAPIMetadata/source/nidmmAPIMetadata/grpc_device/attributes/attributes.hapigen) for an example of how to add these tags.

Reply:
Hmm, actually. I think the attribute below is correctly using this enum 'AddressType'. I don't think this one ever should have... Not sure how we want to deal with this for backwards compatibility. I'd opt to just remove the 'enum' tag like you have here but this will change the .proto.

Reply:
left this as is for now